ztp/gitops-subscriptions/argocd/SecureBoot.md

```diff
@@ -0,0 +1,80 @@
+# Enable and Verify for Secure Boot for SNO
+
+## How to Enable Secure Boot 
+
+ - Configure `SiteConfig` with `UEFIScureBoot` for `bootMode`. E.g part of yaml below 
+ ```yaml
+nodes:
+  - hostName: "myhost"
+    bootMode: "UEFISecureBoot"
+```
+
+- This value should now be available in the Hub (cluster w/ ACM) cluster's appropriate BareMetalHost (bmh) CR. e.g part of the yaml below 
+```yaml
+spec:
+  bootMode: UEFISecureBoot
+```
+
+## How Verify Secure Boot Status
+
+Right now the installation verification is done by `ssh`-ing into the node. Credentials such as private key associated with the node are required.
+E.g `ssh` command 
+```shell
+ssh -i key core@myhost.rh.com
+```
```

Comment:
Another way to do this besides ssh could be `oc debug mode/nodename`; This is maybe a more OCP-feiendly way of getting a shell on a node?

Reply:
Node debug does pick up logs from node's journalctl or dmesg 
```
sh-4.4# journalctl | grep secureboot 
No journal files were found.
````

Reply:
You might need to link up /var/log properly - not sure how that's normally done for 'oc debug'...  You may need to also run `chroot /host` first.

Also, `journalctl -g secureboot` may be slightly more efficient than `journalctl | grep secureboot`

Reply:
You're 100%! The message was right there 😶 

```
[core@cnfdf04 ~]$ oc debug node/cnfdf04.ran.dfwt5g.lab
Starting pod/cnfdf04randfwt5glab-debug ...
To use host binaries, run `chroot /host`
```

`-g` looks good 

```
sh-4.4# journalctl -g secureboot
-- Logs begin at Wed 2022-03-23 17:14:14 UTC, end at Fri 2022-03-25 16:46:26 UTC. --
Mar 23 17:14:14 localhost kernel: secureboot: Secure boot enabled
-- Reboot --
Mar 23 17:20:00 localhost kernel: secureboot: Secure boot enabled
-- Reboot --
Mar 23 17:54:41 localhost kernel: secureboot: Secure boot enabled
-- Reboot --
Mar 23 18:04:16 localhost kernel: secureboot: Secure boot enabled
```

Updating the docs